app/featureset/featureset.go

```diff
@@ -38,12 +38,15 @@ type Feature string
 const (
 	// QBFTConsensus introduces qbft consensus, see https://github.com/ObolNetwork/charon/issues/445.
 	QBFTConsensus Feature = "qbft_consensus"
+	// FrostDKG introduces Frost DKG algorithm into charon.
+	FrostDKG Feature = "frost_dkg"
```

Comment:
My comment might have gotten lost, but where are you going to put this feature flag to switch stuff on and off? Is we are not going to use a feature flag in the code, then there is no need to add something here.